community/modules/scheduler/gke-cluster/README.md

```diff
@@ -1,5 +1,59 @@
+## Description
+
+This module creates a Kubernetes cluster using
+[GKE](https://cloud.google.com/kubernetes-engine).
+
+> **_NOTE:_** This is an experimental module and the functionality and
+> documentation will likely be updated in the near future. This module has only
+> been tested in limited capacity.
+
+### Example
+
+The following example creates a GKE cluster and a VPC designed to work with GKE.
+See [VPC Network](#vpc-network) section for more information about network
+requirements.
+
+```yaml
+  - id: network1
```

Comment:
why not provide a full blueprint? It sounds silly you are omitting 4 lines. Why not just embed teh file? 
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet

Reply:
A full blueprint is an additional 9 lines (11 with white space). To provide a full blueprint would be breaking with the pattern we have on every other module documentation page ([example](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/develop/community/modules/file-system/DDN-EXAScaler/README.md#mounting)). 

I think the idea of an embedded code snippet is interesting but it seems it is quick to cut to a scrolling pane (see example below) instead of showing the full text. Also, I expect that blueprint to evolve a lot as we grow the ecosystem (node pool, job, file systems, etc.). Also, since it has to have a commit reference in the permalink, it takes an additional commit to make updates (one to update the example, another to update the link).

I will start by adding a reference to the gke.yaml example. As for the snippet, my preference would be to leave as is, but I would be open to adding content required to make it a valid blueprint (aka, add the 9 lines to this static snippet). What would you prefer?

https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/062e3d0e3f41ca9b9ea0aaceb216dedbe3d617a0/community/examples/gke.yaml#L27-L40

Reply:
I added a link to the full example but will otherwise leave as is.